backend/api/job.proto

```diff
@@ -199,6 +245,7 @@ message Trigger {
   }
 }
 
+
 message Job {
```

Comment:
In v1 pipeline version is optional (passed as resources reference) for Jobs, it should be included here as an optional input field to allow user to define the pipeline version